docs/content/users/install/docker-installation.md

```diff
@@ -29,22 +29,22 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     2. Install Colima with `brew install colima`.
     3. Start Colima with 4 CPUs, 6GB memory, 100GB storage, and Cloudflare DNS, adjusting as needed:
     ```
-    colima start --cpu 4 --memory 6 --disk 100 --vm-type=qemu --mount-type=sshfs --dns=1.1.1.1
+    colima start --cpu 4 --memory 6 --disk 100 --vm-type=vz --mount-type=virtiofs --dns=1.1.1.1
     ```
-    (On macOS versions before Ventura, omit the `--vm-type=qemu` flag as it doesn't work on older OS versions.)
+    (On macOS versions before Ventura, use `colima start --cpu 4 --memory 6 --disk 100 --mount-type=sshfs --dns=1.1.1.1` as the other options are not supported on older macOS versions.)
 
     After the initial run above, you can use `colima start` or use `colima start -e` to edit the configuration file. Run `colima status` at any time to check Colima’s status.
 
     When your computer restarts, you’ll need to `colima start` again. If you prefer to start Colima automatically on reboot, use `brew services start colima` in Colima version 0.6+ to configure auto-start.
 
     !!!tip "Colima disk allocation"
-        In Colima versions starting with 0.5.4 you can increase—but not decrease—the disk allocation by editing `~/.colima/default/colima.yaml` to change the `disk` setting to a higher value. For example, `disk: 200` will increase allocation to 200 gigabytes. Then `colima restart` will result in the new disk allocation.
+        In Colima you can increase—but not decrease—the disk allocation by editing `~/.colima/default/colima.yaml` to change the `disk` setting to a higher value. For example, `disk: 200` will increase allocation to 200 gigabytes. Then `colima restart` will create the new disk allocation.
 
     !!!warning "Docker contexts let the Docker client point at the right Docker server"
         Colima activates its own Docker context to prevent conflicts with Docker Desktop. If you run `docker context ls`, you’ll see a list of available contexts where the currently-active one is indicated with a `*`—which will be `colima` after you’ve started it. You can change to the default (Docker Desktop) with `docker context use default` or change back with `docker context use colima`. This means you can run Docker Desktop and Colima at the same time, but be mindful of which context you’re pointing at!
 
     !!!warning "Colima can only work in your home directory unless you do further configuration"
```

Comment:
```suggestion
    !!!warning "Without additional configuration, Colima can only provide Docker to DDEV projects located in your home directory"
```
not the perfect headline yet. i just wanted to communicate the gist i try convey. but in the previous version it could be misread that colima only works in the homedirectory but technically they details the colima documentation is about deals with adding additional (writeable) mounts outside of the home directory which is the default